kernelci/cli.py

```diff
@@ -189,6 +189,16 @@ class Args(object):
         'help': "Path the output directory",
     }
 
+    install = {
```

Comment:
Maybe call it `install_path` / `--install-path` to be consistent with `mod_path` and `json_path`, and also be clear it's a path.